Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -31,7 +31,8 @@
     'Sanity Test - Playbook with integration',
     'Sanity Test - Playbook with no integration',
     'Sanity Test - Playbook with mocked integration',
-    'Sanity Test - Playbook with Unmockable Integration',
+    'Sanity Test - Playbook with Unmockable Integration check'
```

Comment:
I'd suggest removing the check, maybe adding `whois` (btw, is it unmockable?) to make it clearer